modules/rhel-compute-updating.adoc

```diff
@@ -34,7 +34,7 @@ your RHEL machines and the `upgrade` playbook.
 +
 [NOTE]
 ====
-You must not enable firewalld later. If you do, you cannot access {product-title} logs on the worker.
+By default, the firewalld service is enabled on the host during installation. Having the firewalld service enabled on your host prevents you from accessing {product-title} logs on the worker. Do not enable firewalld later if you wish to continue accessing {product-title} logs on the worker. 
```

Comment:
The firewalld service would be disabled during RHEL scale-up - https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_node/tasks/config.yml#L22.
So "By default, the firewalld service is enabled on the host during installation" is not correct.

Reply:
@gpei - Is firewalld enabled by default at any point? If not, then I can drop that statement.

Reply:
According to https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_node/tasks/config.yml#L19-L20 
`The base OS RHEL with "Minimal" installation option is enabled firewalld serivce by default`